src/QMCWaveFunctions/Fermion/DelayedUpdateCUDA.h

```diff
@@ -141,11 +141,11 @@ class DelayedUpdateCUDA
   void invert_transpose(const Matrix<T>& logdetT, Matrix<T>& Ainv, std::complex<TREAL>& log_value)
   {
 #if defined(QMC_CUDA2HIP)
-    host_inverter_.invert_transpose(logdetT, Ainv, log_value);
-    initializeInv(Ainv);
+    clearDelayCount();
+    rocsolver_inverter.invert_transpose(logdetT, Ainv, Ainv_gpu, log_value);
 #else
     clearDelayCount();
```

Comment:
Could you move clearDelayCount() out of the ifdef. So we avoid repeating the line.